mos_tests/rabbitmq_oslo/conftest.py

```diff
@@ -38,3 +44,57 @@ def fixt_kill_rpc_server_client(env):
         with node.ssh() as remote:
             cmd = 'pkill -f oslo_msg_check_'
             remote.execute(cmd)
+
+
+@pytest.yield_fixture
+def fixt_iptables_drop_reject(env, request):
```

Comment:
Let's call it like `patch_iptables` (or smth more suitable to real action of fixture)